src/slider/styled-components.js

```diff
@@ -75,88 +75,55 @@ export const TickBar = styled('div', props => {
-  const thumbColor = getThumbColor({...props, $isActive});
-  let toggleSVG = '';
-  const $isStart = $isRange && $index % 2 === 0;
-  const $isEnd = $isRange && $index % 2 !== 0;
-  if ($isStart) {
-    toggleSVG = startThumbIcon(backgroundColor, thumbColor);
-  } else if ($isEnd) {
-    toggleSVG = endThumbIcon(backgroundColor, thumbColor);
-  } else {
-    toggleSVG = singleThumbIcon(backgroundColor, thumbColor);
-  }
+  const {$theme, $values, $thumbIndex, $disabled} = props;
+  const isLeft = $values.length === 2 && $thumbIndex === 0;
+  const isRight = $values.length === 2 && $thumbIndex === 1;
   return {
-    transitionDuration: animation.timing100,
-    transitionTimingFunction: animation.easeOutCurve,
-    transitionProperty: 'background-color',
-    ':after': {
-      position: 'absolute',
-      content: `url('data:image/svg+xml;utf8,${toggleSVG}')`,
-    },
-    position: 'relative',
-    top: '-14px',
-    cursor: 'pointer',
-    zIndex: '1',
-    left: `${offset}%`,
-    marginLeft: $isRange ? '-8px' : $index % 2 ? '-32px' : '-16px',
-    ':before': {
-      position: 'absolute',
-      content: `"${Math.round($value[$index])}"`,
-      top: '-14px',
-      marginLeft: sizing.scale100,
-      width: sizing.scale600,
-      height: sizing.scale600,
-    },
+    height: '24px',
+    width: isLeft || isRight ? '12px' : '24px',
+    borderTopLeftRadius: isRight ? '1px' : '4px',
+    borderTopRightRadius: isLeft ? '1px' : '4px',
+    borderBottomLeftRadius: isRight ? '1px' : '4px',
+    borderBottomRightRadius: isLeft ? '1px' : '4px',
+    backgroundColor: $theme.colors.mono100,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderColor: $theme.colors.mono400,
+    boxShadow: '0px 1px 4px rgba(0, 0, 0, 0.12)',
+    cursor: $disabled ? 'not-allowed' : 'inherit',
   };
 });
 Thumb.displayName = 'StyledThumb';
 
-function getThumbColor(props) {
-  const {
-    $disabled,
-    $isFocused,
-    $isError,
-    $isHovered,
-    $isActive,
-    $theme,
-  } = props;
-  const {colors} = $theme;
-  if ($disabled) {
-    return colors.mono600;
-  } else if ($isActive || $isFocused || $isHovered) {
-    return colors.primary400;
-  } else if ($isError) {
-    if ($isActive || $isFocused) {
-      return colors.negative200;
-    } else if ($isHovered) {
-      return colors.negative100;
-    } else {
-      return colors.negative50;
-    }
-  } else {
-    return colors.mono600;
-  }
-}
+export const InnerThumb = styled('div', props => {
+  const {$theme, $isDragged} = props;
+  return {
+    height: '8px',
+    width: '2px',
+    borderRadius: '2px',
+    backgroundColor: $isDragged ? $theme.colors.primary : $theme.colors.mono600,
+  };
+});
+InnerThumb.displayName = 'StyledInnerThumb';
+
+export const ThumbValue = styled('div', props => {
+  const {$theme} = props;
+  return {
+    position: 'absolute',
+    top: '-24px',
```

Comment:
is this dependent on the font size? should it be calculated so that it can handle customized theme sizing?